config/locales/sv.yml

```diff
@@ -0,0 +1,116 @@
+sv:
+  ice_cube:
+    pieces_connector: ' / '
+    not: 'inte %{target}'
+    not_on: 'inte på %{target}'
+    date:
+      formats:
+        default: '%-d %B, %Y'
+      month_names:
+      -
+      - Januari
+      - Februari
+      - Mars
+      - April
+      - Maj
+      - Juni
+      - Juli
+      - Augusti
+      - September
+      - Oktober
+      - November
+      - December
+      day_names:
+      - Söndag
+      - Måndag
+      - Tisdag
+      - Onsdag
+      - Torsdag
+      - Fredag
+      - Lördag
+    times:
+      other: '%{count} gånger'
+      one: '%{count} gång'
+    until: 'till %{date}'
+    days_of_week: '%{segments} %{day}'
+    days_of_month:
+      other: '%{segments} dagar i månaden'
+      one: '%{segments} dagen i månaden'
```

Comment:
`'%{segments} dag i månaden'`?
`dagen` would work if it's `Första dagen i månaden`, but weird with `En/1 dag i månaden`